pkg/controller/utils/dynatrace_client_reconciler_test.go

```diff
@@ -33,7 +33,7 @@ func TestReconcileDynatraceClient_TokenValidation(t *testing.T) {
 
 	t.Run("No secret", func(t *testing.T) {
 		oa := base.DeepCopy()
-		c := fake.NewFakeClientWithScheme(scheme.Scheme)
+		c := fake.NewFakeClient()
```

Comment:
`fake.NewFakeClient` is deprecated https://godoc.org/sigs.k8s.io/controller-runtime/pkg/client/fake

Reply:
Changed